src/Template/Layout/default.ctp

```diff
@@ -16,7 +20,9 @@ if (!$this->fetch('html')) {
  */
 if (!$this->fetch('title')) {
     $this->start('title');
-    echo Configure::read('App.title');
+    if (Configure::check('App.title')) {
```

Comment:
This check is unnecessary, if title isn't set nothing is going to be output anyway.